### PR TITLE
Makefile: add a 'make ddltest' target to generate ddltest binary

### PR DESCRIPTION
**Pull request by tiancaiamao**
### What problem does this PR solve? 

It's easier to maintenance if we follow these rules:
tidb repo provides ddltest source code and binary 
tidb-test repo provides the CI script

### What is changed and how it works?

Provide `make ddltest` target in the Makefile

### Check List 

Tests 

 - No code

### CODE REVIEW

**Comment by tiancaiamao** [comment]
PTAL @zyguan

File: Makefile [hunk 1]
```diff
@@ -116,6 +116,9 @@ test: checklist checkdep gotest explaintest
 explaintest: server
 	@cd cmd/explaintest && ./run-tests.sh -s ../../bin/tidb-server
 
+ddltest:
+	@cd cmd/ddltest && $(GO) test -o ddltest -c
```
Comment (zyguan) [inline Makefile]: How about `-o ../../bin/ddltest`?

**Pull request merged**